src/views/components/response_components/response.vue

```diff
@@ -18,26 +19,51 @@ import { Survey, Model } from "survey-vue";
 export default {
     name: "response",
     props: ["survey", "view_mode"],
+    data: function() {
+        return {
+            vmode: this.view_mode
+        };
+    },
     methods: {
         //parse the question content and build into a view only question
         parse_question: function(content) {
             let content_object = JSON.parse(content);
-            // change the question type to display the title in html to
-            // avoid displaying inputs for the question
-            let json = {
-                elements: [
-                    {
-                        type: "html",
-                        html: content_object.title
-                    }
-                ]
-            };
+            console.log(content_object);
+            let json;
+            switch (this.view_mode) {
+                case "Student View":
+                    // change the question type to display the title in html to
+                    // avoid displaying inputs for the question
+                    json = {
+                        elements: [content_object]
+                    };
+                    break;
+                case "Question Only":
+                    json = {
+                        elements: [
+                            {
+                                type: "html",
+                                html: content_object.title
+                            }
+                        ]
+                    };
+                    break;
+                default:
+                    console.warn("view mode not recognized");
+            }
+
             let survey = new Model(json);
             // to remove the submit button for survey, set to read-only mode
             survey.mode = "display";
+            survey.showQuestionNumbers = "off";
```

Comment:
Why are you using `"off"` instead of `false`?

Reply:
This is specified in the SurveyJS documentation. The options were `on`, `onPage` and `off`